public/app/plugins/datasource/tempo/_importedDependencies/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains dependencies that we duplicated from Grafana.
+The long-term goal is to move these away from here by replacing them with packages.
+As such, these are only meant to be used internally to this package, they are not intended to be exposed and exported.Expect frequent breaking changes if used externally.
```

Comment:
Bit weird wording here. It seems like somebody outside of grafana is likely to see this as they would try to use this but this seems like an internal directory in a plugin so as such I think `breaking changes` are sort of given as they should not affect any code that will depend on the plugin.

Reply:
Also, what do we need to do to remedy this? Seems like a lot of code that can easily accumulate some drift with changes to core. What do we have to do to remove this as quickly as possible?